databuilder/extractor/neo4j_search_data_extractor.py

```diff
@@ -25,6 +25,7 @@ class Neo4jSearchDataExtractor(Extractor):
         OPTIONAL MATCH (table)-[:COLUMN]->(cols:Column)
         OPTIONAL MATCH (cols)-[:DESCRIPTION]->(col_description:Description)
         OPTIONAL MATCH (table)-[:TAGGED_BY]->(tags:Tag)
```

Comment:
actually if we have a badge, it will show up in tags list as well right

Reply:
yes, i wanted to address that in a second pr after i edited the search service because if remove badges from tags before the search service is updated, you can't search by badge